guide/popular-topics/intents.md

```diff
@@ -100,7 +72,7 @@ myIntents.add('GUILD_PRESENCES', 'GUILD_MEMBERS');
 
 const client = new Client({ ws: { intents: myIntents } });
 
-// more examples of manipulating the bit field
+// more examples about manipulating the bitfield
 
 const otherIntents = new Intents(Intents.NON_PRIVILEGED);
 otherIntents.remove(['GUILDS', 'GUILD_MESSAGES']);
```

Comment:
That's not a line you changed, but it may be good to _not_ remove GUILDS, even in an example, since that'll break everything guild-related. (read: everything but DMs)

```suggestion
otherIntents.remove(['DIRECT_MESSAGES', 'GUILD_MESSAGES']);
```
(Or pretty much any other intent)